shared/js/default_activity_helper.js

```diff
@@ -0,0 +1,151 @@
+/* global SettingsHelper */
```

Comment:
I'm gonna need it for the settings part (bug 1039386), so I thought it wouldn't hurt to place it on `shared` from the first moment.